src/test/ui/associated-consts/assoc-const.rs

```diff
@@ -13,9 +15,8 @@ impl Foo for Bar {
 const TEST:usize = 3;
 
 
-fn foo<F: Foo<N=3>>() {}
-//~^ ERROR associated const equality is incomplete
-fn bar<F: Foo<N={TEST}>>() {}
-//~^ ERROR associated const equality is incomplete
+fn foo<F: Foo<N=3usize>>() {}
```

Comment:
what happens on the previous two tests? Unless they are tested elsewhere, maybe leave them in for test coverage?

Reply:
ah yea I just simplified it for now, will add back later.